cmd/ebrelayer/txs/parser.go

```diff
@@ -68,14 +68,14 @@ func EthereumEventToEthBridgeClaim(valAddr sdk.ValAddress, event types.EthereumE
 	nonce := int(event.Nonce.Int64())
 
 	// Package the information in a unique EthBridgeClaim
-	witnessClaim.EthereumChainID = chainID
-	witnessClaim.BridgeContractAddress = bridgeContractAddress
-	witnessClaim.Nonce = nonce
-	witnessClaim.TokenContractAddress = tokenContractAddress
+	witnessClaim.EthereumChainId = int64(chainID)
+	witnessClaim.BridgeContractAddress = bridgeContractAddress.String()
+	witnessClaim.Nonce = int64(nonce)
```

Comment:
Same thing with nonce. It should just be an int.

Reply:
Did we do this because protobuffs do not support regular ints? cc @timlind 
https://developers.google.com/protocol-buffers/docs/proto3#scalar